Python/compile.c

```diff
@@ -9056,7 +9090,7 @@ swaptimize(basicblock *block, int *ix)
     // our "stack":
     for (int i = 0; i < len; i++) {
         if (instructions[i].i_opcode == SWAP) {
-            int oparg = instructions[i].i_oparg;
+            int oparg = OPARG_VALUE(instructions[i].i_oparg);
```

Comment:
should we also assert the `EXPLICIT_ARG` type here?

or more generally, should `OPARG_VALUE` take an oparg type also, and in debug builds always assert the expected type?